server/graphql/stock_line/src/lib.rs

```diff
@@ -46,6 +46,7 @@ pub struct StockLineFilterInput {
     pub id: Option<EqualFilterStringInput>,
     pub is_available: Option<bool>,
     pub item_code_or_name: Option<StringFilterInput>,
+    pub search: Option<StringFilterInput>,
```

Comment:
Adding search, leaving item_code_or_name in place, as it is used in report filters.